src/hssm/plotting/model_cartoon.py

```diff
@@ -0,0 +1,1985 @@
+                ]
+            ),
+            np.min(
+                [
+                    np.minimum(
+                        (-1) * posterior_pred_no_noise[key_]["metadata"]["boundary"], 0
+                    )[0]
+                    for key_, _ in posterior_pred_no_noise.items()
+                ]
+            ),
+        )
+        hist_bottom_high = kwargs.get("hist_bottom_high", b_high)
+        hist_bottom_low = kwargs.get("hist_bottom_low", -b_low)
+    else:
+        _logger.warning(
+            'No "theta_mean" provided. Using default values for histogram'
+            " location. Likely highly suboptimal choice!"
+        )
+        hist_bottom_high = 3
+        hist_bottom_low = 3
+
+    hist_histtype = kwargs.get("hist_histtype", "step")
+
+    axis.set_xlim(xlim_low, xlim_high)
+    axis.set_ylim(ylim_low, ylim_high)
+    axis_twin_up: Axes = cast(Axes, axis.twinx())
+    axis_twin_down: Axes = cast(Axes, axis.twinx())
+    axis_twin_up.set_ylim(ylim_low, ylim_high)
+    axis_twin_up.set_yticks([])
+    axis_twin_down.set_ylim(ylim_high, ylim_low)
+    axis_twin_down.set_yticks([])
+    axis_twin_down.set_axis_off()
+    axis_twin_up.set_axis_off()
+
+    # This ensures zorder across axis elements is correct
+    # for the sequence in which they are invoked.
+    axis.set_zorder(1)
+    axis.set_facecolor("none")
+    axis_twin_up.set_zorder(0)
+    axis_twin_down.set_zorder(0)
+
+    if theta_mean is not None:
+        weights_up = np.tile(
+            (1 / bin_size) / sim_out["rts"][(sim_out["rts"] != -999)].shape[0],
+            reps=sim_out["rts"][
+                (sim_out["rts"] != -999) & (sim_out["choices"] == 1)
+            ].shape[0],
+        )
+        weights_down = np.tile(
+            (1 / bin_size) / sim_out["rts"][(sim_out["rts"] != -999)].shape[0],
+            reps=sim_out["rts"][
+                (sim_out["rts"] != -999) & (sim_out["choices"] != 1)
+            ].shape[0],
+        )
+
+        # Add histograms for posterior mean simulation
+        axis_twin_up.hist(
+            np.abs(
+                sim_out["rts"][(sim_out["rts"] != -999) & (sim_out["choices"] == 1)]
+            ),
+            bins=bins,
+            weights=weights_up,
+            histtype=hist_histtype,
+            bottom=hist_bottom_high,
+            alpha=1,
+            color=color_pp_mean,
+            edgecolor=color_pp_mean,
+            linewidth=linewidth_histogram,
+            zorder=-1,
+        )
+
+        axis_twin_down.hist(
+            np.abs(
+                sim_out["rts"][(sim_out["rts"] != -999) & (sim_out["choices"] != 1)]
+            ),
+            bins=bins,
+            weights=weights_down,
+            histtype=hist_histtype,
+            bottom=hist_bottom_low,
+            alpha=1,
+            color=color_pp_mean,
+            edgecolor=color_pp_mean,
+            linewidth=linewidth_histogram,
+            zorder=-1,
+        )
```

Comment:
Here the common settings could be extracted or reuse the values suggested earlier to make code a bit more DRY.